cloudinit/cmd/devel/parser.py

```diff
@@ -29,7 +29,8 @@ def get_parser(parser=None):
         ),
         (
             "schema",
-            "Validate cloud-config files for document schema",
+            "schema has been promoted from devel up to a top level subcommand,"
+            " invoking via `cloud-init devel schema` is now deprecated",
```

Comment:
We have a warning with our devel commands that they can and will change (and likely be promoted top-level), so I'm ok removing this entirely rather than deprecating it.